src/raftstore/store/worker/apply.rs

```diff
@@ -231,13 +238,13 @@ impl ApplyCallback {
 
 /// Stash keeps the informations that are needed to restore an appropriate
 /// applying context for the `ApplyContextCore::stash` call.
-struct Stash {
+pub struct Stash {
```

Comment:
`Stash` and `ApplyContext` don't need to be public. What we need to do is change `pub fn handle_reft_committed_entries` to private.